lib/azure/classes.ts

```diff
@@ -0,0 +1,55 @@
+import { AzureadProvider } from '@cdktf/provider-azuread/lib/provider';
+import { AzurermProvider } from '@cdktf/provider-azurerm/lib/provider';
+import { AzurermBackend } from 'cdktf/lib/backends/azurerm-backend';
+import { Construct } from 'constructs';
+import { EnvProps } from './interfaces';
+
+export enum AzureProviders {
+  AZURE = 'azure_provider',
+  AZUREAD = 'azure_ad_provider'
+
+}
+
+export interface AzureConstructProps {
+  isLocalBackend?: boolean;
+  provider?: AzureProviders;
+  env: EnvProps;
+  tags?: Record<string, string>;
```

Comment:
```suggestion
  tags?: Record<string, string | number>;
```
there some cases when need to specify number, like `kubernetes.io/cluster/internal-lb: 1`